libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3917,6 +3917,30 @@ void GCS_MAVLINK::send_mount_status() const
     mount->send_mount_status(chan);
 }
 
+void GCS_MAVLINK::send_set_position_target_global_int(uint8_t target_system, uint8_t target_component, const Location& loc)
+{
+
+    uint16_t type_mask = POSITION_TARGET_TYPEMASK_VX_IGNORE | POSITION_TARGET_TYPEMASK_VY_IGNORE | POSITION_TARGET_TYPEMASK_VZ_IGNORE | \
```

Comment:
This should probably be -1 (ignore everything), in case more IGNORE options get added, followed by masking in the bits you don't want ignored.

Reply:
I think I might skip over this suggestion.  It's pretty unlikely new bits will ever be added to the message and this is consistent with how we do it in other places..